engine/resources/lua/SphereCreator.lua

```diff
@@ -91,6 +91,7 @@ function SphereCreator:makeSphereMesh(c, r)
     -- vbo:bless();
     local mesh = MOAIMesh.new();
     mesh:setVertexBuffer(vbo, vertexFormat);
+    mesh:setTotalElements(vbo:countElements(vertexFormat))
     mesh:setPrimType(MOAIMesh.GL_TRIANGLES);
     return mesh;

```

Comment:
Nope. 
I think this SphereCreator isn't even used in these graphics tests. This was leftover from some other thing that got muddled into the engine at one point.